tests/tests_default.yml

```diff
@@ -1,5 +1,8 @@
 - name: Ensure that the role runs with default parameters
   hosts: all
+  vars:
+    tests_role_path: "../../timesync"
```

Comment:
`../../timesync` seems to be the git working copy here. IMHO it is wrong to assume that it has a certain name since developers can clone it/name it differently. Also, I guess this would not work when the role is installed via RPM. Not sure if I really understand what is going on, here.

Reply:
`tests_role_path` variable was discarded and replaced by `{{ '..' | realpath }}` in https://github.com/linux-system-roles/timesync/pull/72/commits/02caf146bacdb9018f241d5c69b3960584571c91. With this improvement, tests should be able to run upstream. To make them working also downstream, when the role is installed via RPM, more sophisticated method is needed.

Reply:
Using the original name like `linux-system-roles.timesync` and a corresponding symlink in the `roles/` dir would work. This is what the role currently does:
https://github.com/linux-system-roles/timesync/tree/master/tests/roles

Why is path needed, here?

The downstream test only needs to remove/not ship this symlink.